mailbox/api/src/main/java/org/apache/james/mailbox/MailboxManager.java

```diff
@@ -328,4 +329,18 @@
      */
     List<MailboxPath> list(MailboxSession session) throws MailboxException;
 
+    /*
```

Comment:
as a general rule, we don't like comments. We should encode as much knowledge as possible into the code (and git) so comments become useless.